tests/js/spec/views/organizationGroupDetails/groupEventDetailsContainer.spec.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import OrganizationEnvironmentsStore from 'app/stores/organizationEnvironmentsStore';
+import GroupEventDetailsContainer from 'app/views/organizationGroupDetails/groupEventDetailsContainer';
+
+jest.mock('app/views/organizationGroupDetails/groupEventDetails', () => ({
+  GroupEventDetails: () => <div>GroupEventDetails</div>,
+}));
+
+describe('groupEventDetailsContainer', () => {
+  const org = TestStubs.Organization();
+  const globalSelection = TestStubs.GlobalSelection();
+
+  beforeEach(() => {
+    OrganizationEnvironmentsStore.init();
+  });
+
+  it('fetches environments', async function() {
+    MockApiClient.addMockResponse({
```

Comment:
This returns a `jest.fn` - so you could additionally assert that it has been called (not as useful in this case, since you test that GroupEventDetails exists, which covers this case).